libntfs-3g/win32_io.c

```diff
@@ -134,6 +127,30 @@ static LPFN_SETFILEPOINTEREX fnSetFilePointerEx = NULL;
 #define FNPOSTFIX "A"
 #endif
 
+/* 
```

Comment:
Is it possible to just skip this ``enum`` when any constant is already defined (instead of doing this bunch of ``undef``'s)?